types/materialize-css/index.d.ts

```diff
@@ -641,6 +642,73 @@ declare global {
          */
         function toast(options: Partial<ToastOptions>): Toast;
 
+        class Collapsible extends Component<CollapsibleOptions> {
+            /**
+             * Get Instance
+             */
+            static getInstance(elem: Element): Collapsible;
+
+            /**
+             * Open Collapsible
+             */
+            open(): void;
+
+            /**
+             * Close Collapsible
+             */
+            close(): void;
+
+            /**
+             * Destroy Collapsible
+             */
+            destroy(): void;
+        }
+
+        interface CollapsibleOptions {
+            /**
+             * If accordion versus collapsible.
+             * @default true
+             */
+            accordion: boolean;
+
+            /**
+             * Callback function called before collapsible is opened.
+             */
+          onOpenStart: () => void;
+
+          /**
+           * Callback function called after collapsible is opened.
+           */
+          onOpenEnd: () => void;
+
+          /**
+           * Callback function called before collapsible is closed.
+           */
+          onCloseStart: () => void;
+
+          /**
+           * Callback function called after collapsible is closed.
+           */
+          onCloseEnd: () => void;
```

Comment:
These callbacks will be called with `this` set to the `Collapsible` object and the first parameter set to the related `Element` object. Although this is not in the document, I think this is very useful and important, and should be included in the definition.